aiida/cmdline/commands/cmd_group.py

```diff
@@ -79,7 +79,7 @@ def group_delete(group, force):
     group_pk = group.pk
     group_name = group.name
 
-    num_nodes = len(group.nodes)
```

Comment:
This should go back to `len(group.nodes)` once the suggested fix for __len__ is in place, for efficiency

Reply:
Done.